Session9/src/com/itschool/homework/Session9Challenge4.java

```diff
@@ -0,0 +1,26 @@
+package com.itschool.homework;
+
+public class Session9Challenge4 {
+    public static void main(String[] args) {
+        Session9Challenge4 run = new Session9Challenge4();
+        run.challengeRun4();
+    }
+
+    public void challengeRun4() {
+        StringBuilder remove = new StringBuilder("Let's go for a ride");
+        System.out.println(remove);
+        for (int i = 0; i < remove.length(); i++) {
+            char ch = remove.charAt(i);
+            if (isVowel(ch)) {
+                remove.deleteCharAt(i);
+                i--;
```

Comment:
I don't get it. What is the purpose of this decrement? Removing it has the same outcome.